src/v2/Apps/Auction/Routes/__tests__/ConfirmBid.jest.tsx

```diff
@@ -574,7 +574,7 @@ describe("Routes/ConfirmBid", () => {
         context_page: AnalyticsSchema.PageName.AuctionConfirmBidPage,
         auction_slug: "saleslug",
         artwork_slug: "artworkslug",
-        bidder_id: null,
```

Comment:
Interesting – did you just manually spot this or was there a type here after the refactoring? I'm still learning more about the semantic differences between `null` and `undefined`

Reply:
There might be subtle differences in the JS world that I'm not aware of, but there's almost no difference between them once it's serialized to JSON and sent over to Segment. The presence of the key should not make any breaking changes.

```ts
JSON.stringify({ foo: null, bar: undefined })
// => '{"foo":null}'
```